### DISCUSSION TITLE
Template for "write to pdf"

### DISCUSSION THREAD

**Discussion by amvFrontendMonkey** [RyotaUshio/obsidian-pdf-plus#301] [Q&A]
Unless i'm misunderstanding, it isn't currently possible to change the text that is written into the pdf when the experimental pdf editing is chosen. I hoped to see possibilities to add (templater) templates for this scenario

My use case is that i would dearly love to select the "save to pdf" and highlight a word. A template would be invoked, in which i have code to do a language translation and return both the original word + a note with the translation. This would be saved into the PDF

**Comment by elmpp** [comment]
@RyotaUshio
